docs/generation.rst

```diff
@@ -3,122 +3,91 @@
+TypeScript/JavaScript.
 
 To get started with generators, first initialize an :ref:`EdgeDB project
-<ref_guide_using_projects>` in the root of your application. Generators will look for an ``edgedb.toml`` file to determine the root of your application.
-
-.. note::
-
-  Follow the :ref:`Quickstart <ref_quickstart>` for detailed instructions on installing the CLI, initializing a project, writing a basic schema, and executing your first migration.
-
-The query builder works for both JavaScript and TypeScript users, and supports both Node.js and Deno. It's possible to use the query builder with or without TypeScript. Some requirements apply to TypeScript users only.
-
-**Node.js**
-- ``Node.js v14+``. Run ``node --version`` to see your current version.
-
-**Deno**
-
-- ``Deno v0.20+``.
-
-**TypeScript**
-
-- ``TypeScript v4.4+``.
-- *For Node.js users*: ``npm install @types/node -D``.
-- Strict mode: ``"strict": true``
-- Downlevel iteration (query builder only): ``"downlevelIteration": true``
-
-.. code-block:: javascript
-
-  // tsconfig.json
-  {
-    // ...
-    "compilerOptions": {
-      // ...
-      "strict": true,
-      "downlevelIteration": true,
-    }
-  }
-
-Installation
-^^^^^^^^^^^^
+<ref_guide_using_projects>` in the root of your application. Generators will
+look for an ``edgedb.toml`` file to determine the root of your application. See
+the :ref:`Overview <edgedb-js-installation>` page for details on installing
 
-.. note::
-
-  If you're using Deno, you can skip this step.
-
-Install the ``edgedb`` package.
-
-.. code-block:: bash
-
-  $ npm install edgedb       # npm users
-  $ yarn add edgedb          # yarn users
-
-Then install ``@edgedb/generate`` as a dev dependency.
+Run a generator with the following command.
 
-.. code-block:: bash
+.. tabs::
 
-  $ npm install @edgedb/generate --save-dev      # npm users
-  $ yarn add @edgedb/generate --dev              # yarn users
+  .. code-tab:: bash
+    :caption: npm
 
-Running a generator
-^^^^^^^^^^^^^^^^^^^
+    $ npx @edgedb/generate <generator> [options]
 
-Run a generator with the following command.
+  .. code-tab:: bash
+    :caption: yarn
 
-**Node.js**
+    $ yarn run -B generate <generator> [options]
 
-.. code-block:: bash
+  .. code-tab:: bash
+    :caption: pnpm
 
-  $ npx @edgedb/generate <generator> [options]
+    $ pnpm exec generate <generator> [options]
 
+  .. code-tab:: bash
+    :caption: Deno
 
-**Deno**
+    $ deno run \
+      --allow-all \
+      --unstable \
+      https://deno.land/x/edgedb/generate.ts <generator> [options]
 
-.. code-block:: bash
+The value of generators should be one of the following:
 
-  $ deno run --allow-all --unstable https://deno.land/x/edgedb/generate.ts <generator> [options]
+.. list-table::
+   :class: funcoptable
 
+   * - ``edgeql-js``
+     - Provides a **code-first** way to write **fully-typed** EdgeQL queries
+       with TypeScript. We recommend it for TypeScript users, or anyone who
```

Comment:
I would also be explicit that this is generating the query builder. The other two seem obvious to me since the names match up, but a fresh user might otherwise not connect this one to the query builder.

Something like "Generates the query builder which provides a code-first…"